apps/expressApps/apps.ts

```diff
@@ -0,0 +1,26 @@
+import express from 'express';
+import cors from 'cors';
+import dragonsController from "./routes/dragonsController";
+import monstersController from "./routes/monstersController";
+import helmet from 'helmet';
+const app = express()
+
+
+const port = 3000; // why 3000??
```

Comment:
this port is up to you to choose. As long as it doesn't conflict with your front end servers, like your lives and hosts, then it is all good.